### ISSUE TITLE
State refresh with real infrastructure / policy name uniqueness / reassign policy attachment

### ISSUE THREAD

**Issue by kkailasababu-talend** [banyansecurity/terraform-provider-banyan#106]
Dear Banyan, 
We have noticed below issues while working on the terraform banyan provider, could you please advice?

1. Banyan terraform will not refresh the state with original infrastructure. When we delete a web application manually from GUI or banyan cli and when  we run a plan afterwards, terraform banyan completely relies on the state file and shows like the deleted application exists. So we need to remove it from state using terraform state rm commands and rerun the plan.
2. Terraform provider should not allow to create policy if the policy exists with same name. It's a high risk.
This will show likes the terraform plans switches between policy contents after each apply without change the code.
Two policies with same name will switch each other during each apply. Would be great if banyan or banyan terraform allows only unique names for policies.
3. Cannot reassign the policy of web services to a new one.
May be it is because of the provider breaking change from 0.x.x to 1.0.0 and change in attribute names.
So basically resources created with old major version cannot be completely manage with new major version, any upgrade guides along with list of complete breaking changes in attribute names or anything similar.

**Comment by nareshkakubal** [comment]
All issues are addressed with later releases. Issue number 3 should work fine between 1.x.x versions. Please feel free to reopen this ticket or create new ticket if any related issue is observed.
~Banyan
